WalletWasabi.Fluent/CrashReport/ViewModels/CrashReportWindowViewModel.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.IO;
+using ReactiveUI;
+using System.Windows.Input;
+using WalletWasabi.Fluent.ViewModels.HelpAndSupport;
+using WalletWasabi.Models;
+using WalletWasabi.Gui.ViewModels;
```

Comment:
Do not refer to the GUI. I assume it is here because of the ViewModelBase, but that is also found in the Fluent project.